index.html

```diff
@@ -293,12 +293,9 @@ <h1 lang="en">Contribute Hardware</h1>
     <div class="design-system">
       <h3 class="strong-title wow fadeInDown" lang="en">openSUSE Design System</h3>
       <h4 class="strong-title wow fadeInDown" data-wow-delay=".5s" lang="en">We've created a Design System to help you contribute consistent web tools and features using the openSUSE colors, writing guides, and more. Check it out:</h4>
-      <a href="http://opensuse.eosdesignsystem.com/" target="_blank" class="btn btn-default wow zoomIn">
+      <a href="https://github.com/openSUSE/chameleon" target="_blank" class="btn btn-default wow zoomIn">
```

Comment:
It's kind of hard to say if this, or the more generic branding guidelines would be a better fit here. https://opensuse.github.io/branding-guidelines/

Reply:
Thanks for the input, didn't know that page was a thing - I think these branding guidelines would be a good fit if they contained a reference to the repository, or did I miss one?